tests/tests.js

```diff
@@ -116,6 +116,10 @@ describe('react-gettext-parser', () => {
       expect(messages).to.have.length(1);
     });
 
+    it('should not fail on import()', () => {
+      const messages = extractMessagesFromFile('tests/fixtures/DynamicImportCaller.js');
```

Comment:
Surely can adjust the description, but I'm not sure the actual import() statement can be tested via a message assertion. It's only a stage 3 proposal for now (requires babel to use). For this specific case, it should be sufficient test-wise to verify that the babylon parser doesn't choke on the syntax. You can remove the babylon plugin `dynamicImport` in the defaults file to see what I'm talking about.